usr_merge.cc

```diff
@@ -19,19 +19,25 @@ static bool check_link(const string& path, const bool mandatory)
 string usr_merge(const string& path)
 {
 	static bool setup = false;
-	static bool MERGED_BIN, MERGED_LIB, MERGED_LIB32, MERGED_SBIN;
+	static bool MERGED_BIN, MERGED_LIB, MERGED_LIB32, MERGED_LIB64, MERGED_LIBO32, MERGED_LIBX32, MERGED_SBIN;
 	if (!setup)
 	{
 		MERGED_BIN=check_link("/bin", true);
 		MERGED_LIB=check_link("/lib", true);
 		MERGED_LIB32=check_link("/lib32", false);
+		MERGED_LIB64=check_link("/lib64", false);
+		MERGED_LIBO32=check_link("/libo32", false);
+		MERGED_LIBX32=check_link("/libx32", false);
 		MERGED_SBIN=check_link("/sbin", true);
 		setup=true;
 	}
 
 	if ((MERGED_BIN and path.rfind("/bin/", 0) == 0)
 	 or (MERGED_LIB and path.rfind("/lib/", 0) == 0)
 	 or (MERGED_LIB32 and path.rfind("/lib32/", 0) == 0)
+	 or (MERGED_LIB64 and path.rfind("/lib64/", 0) == 0)
```

Comment:
I don't know if this is needed for real. 
(and the cpu cost)
Maybe are the existing static liste in rules/ enough ?
We could get a list of all possible path from the Debian Source project or somewhere else.